DataConnectors/M365Defender-VulnerabilityManagement/functionPackage/GetMDVMData/function.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
Hi @anders-alex please remove the whitespace for fixing indent errors in validation checks, thanks!

Reply:
Hi @v-vdixit , I am not clear on what is wrong with this json format or where the whitespace is? I ran through a json validator and it passed without issue. Thanks.

Reply:
Hi @anders-alex the extra whitespace is there in the very first line of 'DataConnectors/M365Defender-VulnerabilityManagement/functionPackage/GetMDVMData/function.json' file, before the '{', please update this file here to fix failing validations, thanks!

Reply:
@v-vdixit - sorry, I am not seeing any whitespace before the "{" on line one:

{
  "bindings": [
    {
      "name": "Timer",
      "type": "timerTrigger",
      "direction": "in",
      "schedule": "0 0 0 * * *"
    }
  ]
}

Reply:
Hi @anders-alex I wanted to let you know that I made this small change for removal of whitespace in the PR you submitted. I apologize for any inconvenience this may have caused and wanted to make sure it was okay with you, thanks!

Reply:
Thanks, @v-vdixitm, no problem at all. Still curious where the actual whitespace was but as long as things are working now I guess it's all good.